src/page/login/login.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {Props} from 'react';
+import {View, Text, TextInput, TouchableHighlight} from 'react-native';
+import {styles} from './loginCss';
+import Reactotron from 'reactotron-react-native';
+// import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+// import {faCoffee} from '@fortawesome/free-solid-svg-icons';
+// import Logo from '../../resource/images/logo.svg';
+
+const Login: React.FC = (Props: Props) => {
+  /**
+   * login
+   */
+  function login() {
+    //https://reactnative.dev/movies.json
+    return (
+      fetch('https://reactnative.dev/movies.json', {
+        // method: 'POST',
+        // headers: {
+        //   Accept: 'application/json',
+        //   'Content-Type': 'application/json',
+        // },
+        // body: {
+        //   loginname: 'jerry_liu',
+        //   password: '1qaz@WSX',
+        //   callback: () => {
+        //     Reactotron.log('123');
+        //   },
+        // },
+      })
+        .then(response => {
+          Reactotron.log(response);
+          response.json();
+        })
+        // .then(json => {
+        //   Reactotron.log(json);
+        //   return json.movies;
+        // })
+        .catch(error => {
+          console.error(error);
+        })
+    );
+  }
+  return (
+    <View style={styles.container}>
```

Comment:
This view is not corresponding to the design. Please add gradient for background, reduce bottom offset and add 
[KeyboardAvoidingView](https://reactnative.dev/docs/keyboardavoidingview) to adjust inputs to keyboard